benchmarks/distributed/rpc/rl/coordinator.py

```diff
@@ -72,7 +72,7 @@ def run_coordinator(self, episodes, episode_steps, queue):
             print(f"Episode {ep} - ", end="")
 
             n_steps = episode_steps
-            agent_start_time = time.time()
+            agent_start_time = time.time()  # noqa: F841
```

Comment:
Sounds like a bug right?
cc @wconstab is this code still in use?

Reply:
Well, it basically duplicates the previous `time.time` which only has a `print` statement separating them, so I think it's to be deleted, so I did.